ComfySigns/Patches/ZoneSystemPatch.cs

```diff
@@ -0,0 +1,20 @@
+﻿using HarmonyLib;
+
+using ComfyLib;
+
+using static ComfySigns.ComfySigns;
+using static ComfySigns.PluginConfig;
+
+namespace ComfySigns.Patches {
```

Comment:
Just default to regular ComfySigns namespace.

I researched this before and C# namespace practices aren't the same as Java (where the package matches the folder structure).